src/webextension/widgets/panel.js

```diff
@@ -11,9 +11,13 @@ import styles from "./panel.css";
 
 export function PanelHeader({className, onBack, children}) {
   const finalClassName = `${styles.panelHeader} ${className}`.trimRight();
+  const imgSrc = browser.extension.getURL("/icons/arrowhead-left-16.svg");
+  // TODO: actually localize this string
+  const imgAlt = "gO bACk";
```

Comment:
Don't forget to localize this.

Reply:
>Don't forget to localize this.

@jimporter I'm not sure how. If you see my prior commit at 9db70b2 it returns "[object Object]" and I'm not sure how to get just the string out of it. Is this something you can help with?

Reply:
You'll want something like the following:

widgets.ftl:
```
panel-go-back-button
  .alt = Go Back
```

panel.js:
```js
<Button theme="ghost" size="micro" onClick={onBack}>
  <Localized id="panel-go-back-button">
    <img src={imgSrc}/>
  </Localized>
</Button>
```

Reply:
(done)